src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  font-family: "Roboto", Arial, sans-serif;
+  font-style: normal;
+
+  &__image {
+    display: block;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 40px 0 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $card-text-color-accent;
+    transition: color $card-transition-duration;
+  }
+
+  &:hover &__title {
+    color: #34568b;
+  }
```

Comment:
```suggestion
    transition: color $card-transition-duration;
    &:hover {
    color: #34568b;
  }
  }
```

Reply:
...- change the card title text color to `#34568b` when the card is hovered (`.card:hover .card__title`)...

Reply:
that's what @lerastarynets mean:
```scss
&__title {
    margin: 40px 0 0;
    font-size: 12px;
    font-weight: 500;
    line-height: 18px;
    color: $card-text-color-accent;
    transition: color $card-transition-duration;

    &:hover  {
        color: #34568b;
    }
}
```

Reply:
it's not what in the requirement to task. now color will be changed only if you point to title(not card).